sources/packages/api/src/database/entities/disbursement-receipts.model.ts

```diff
@@ -0,0 +1,214 @@
+   */
+  @Column({
+    name: "funding_type",
+    nullable: false,
+  })
+  fundingType: string;
+
+  /**
+   * Total entitled disbursed amount for either FE(Federal) or BC(Provincial)..
+   */
+  @Column({
+    name: "total_entitled_disbursed_amount",
+    type: "numeric",
+    nullable: false,
+  })
+  totalEntitledDisbursedAmount: string;
+
+  /**
+   * Total disbursed amount for either FE(Federal) or BC(Provincial)..
+   */
+  @Column({
+    name: "total_disbursed_amount",
+    type: "numeric",
+    nullable: false,
+  })
+  totalDisbursedAmount: string;
+
+  /**
+   * Financial posting date of NSLSC.
+   */
+  @Column({
+    name: "disburse_date",
+    type: "date",
+    transformer: dateOnlyTransformer,
+    nullable: false,
+  })
+  disburseDate: Date;
+
+  /**
+   * Amount disbursed to the student.
+   */
+  @Column({
+    name: "disburse_amount_student",
+    type: "numeric",
+    nullable: false,
+  })
+  disburseAmountStudent: string;
+
+  /**
+   * Amount disbursed to the institution.
+   */
+  @Column({
+    name: "disburse_amount_institution",
+    type: "numeric",
+    nullable: false,
+  })
+  disburseAmountInstitution: string;
+
+  /**
+   * The date signed by the institution of the student at the time they receive the loan certificate.
+   */
+  @Column({
+    name: "date_signed_institution",
+    type: "date",
+    transformer: dateOnlyTransformer,
+    nullable: false,
+  })
+  dateSignedInstitution: Date;
+
+  /**
+   * Institution code.
+   */
+  @Column({
+    name: "institution_code",
+    nullable: false,
+  })
+  institutionCode: string;
+
+  /**
+   * Method of disbursement to student. C(By cheque),E(By EFT), <space> disbursement only to institution.
+   */
+  @Column({
+    name: "disburse_method_student",
+    nullable: false,
+  })
+  disburseMethodStudent: string;
+
+  /**
+   * Total Federal or BC grant entitled amount.
+   */
+  @Column({
+    name: "total_entitled_grant_amount",
+    type: "numeric",
+    nullable: false,
+  })
+  totalEntitledGrantAmount: string;
+
+  /**
+   * Total Federal or BC grant disbursed amount.
+   */
+  @Column({
+    name: "total_disbursed_grant_amount",
+    type: "numeric",
+    nullable: false,
+  })
+  totalDisbursedGrantAmount: string;
+
+  /**
+   * Total Federal or BC grant disbursed amount to student.
+   */
+  @Column({
+    name: "total_disbursed_grant_amount_student",
+    type: "numeric",
+    nullable: false,
+  })
+  totalDisbursedGrantAmountStudent: string;
+
+  /**
+   * Total Federal or BC grant disbursed amount to institution.
+   */
+  @Column({
+    name: "total_disbursed_grant_amount_institution",
+    type: "numeric",
+    nullable: false,
+  })
+  totalDisbursedGrantAmountInstitution: string;
+
+  /**
+   * Values for this disbursement.
+   */
+  @OneToMany(
+    () => DisbursementReceiptValue,
+    (disbursementReceiptValue) => disbursementReceiptValue.disbursementReceipt,
+    {
+      eager: true,
+      cascade: true,
```

Comment:
I would recommend allowing the cascade only for `insert` here. The records are never going to be updated or removed as far as we know so far, right?